scripts/changelog/.env

```diff
@@ -0,0 +1,7 @@
+RUSTC_STABLE="rustc 1.56.1 (59eed8a2a 2021-11-01)"
+RUSTC_NIGHTLY="rustc 1.57.0-nightly (51e514c0f 2021-09-12)"
+PRE_RELEASE=true
+HIDE_SRTOOL_ROCOCO=true
+HIDE_SRTOOL_SHELL=true
+REF1=statemine-v5.0.0
+REF2=HEAD
```

Comment:
This should be checked in?

Reply:
It would not have hurt but I also thought it could be removed. It is gone.